es_managers/es_idx_manager/app.py

```diff
@@ -144,8 +144,8 @@ def should_delete_index(self, index):
             boolean: boolean for whether index should be deleted
         """
         should_delete = False
-        index_name = '-'.join(word for word in index["index"].split("-")[:-1])
-        index_date = index["index"].split("-")[-1]
+        index_name = '-'.join(word for word in index["index"].split("-", 1)[:-1])
+        index_date = index["index"].split("-", 1)[-1]
```

Comment:
I think this will cause the index manager to fail for the current indices which do not follow the new pattern.

Reply:
All the indices currently on elastic search following the new dash naming convention so should not fail out

Reply:
Ah true. It might cause some issues with the tests in travis, however. Earlier this week I implemented different index prefixes in test to prevent concurrent es_manager tests from failing: https://github.com/HumanCellAtlas/logs/blob/master/es_managers/es_idx_manager/test/test_app.py#L11-L14